templates/Makefile

```diff
@@ -40,7 +41,7 @@ apply-k8s-utils:
 	terraform apply $(AUTO_APPROVE)
 
```

Comment:
We will need a new make target to apply the shared environment.

Reply:
done apply-shared-env prior to apply-env